llorens/day03/Day03Solver.py

```diff
@@ -0,0 +1,86 @@
+def calculate_number(matrix_indexes, x, y):
+    res = 0
+    for i in range(-1, 2):
+        for j in range(-1, 2):
+            try:
+                res += matrix_indexes[(x+i, y+j)]
+            except:
+                res += 0
+
+    return  res
```

Comment:
Nice way of calculating neighbors sum. It is so clear 💯

Reply:
Yes! but I don't like the `try` `except`  sooooo slow but   neighbors only has 3 or 4